### DISCUSSION TITLE
sio.emit I not control

### DISCUSSION THREAD

**Discussion by isdito** [miguelgrinberg/python-socketio#1125] [Q&A]
Hello,

The sdk say me this 
Default IP
The default IP for the CamFi is 192.168.9.67 or 192.168.1.67. Take the photo
GET takepic/true
Trigger the camera to take a photo

I Connect 

```
sio = socketio.Client()
sio.connect('http://192.168.9.67:8080')
```

But the emit not works

```
def my_event(sid, data):
    # handle the message
    print ("ddddddddddddffffff")
    return "OK", 123
    
sio.emit('my_event','GET takepic/true')
```
if you dont mind heko me

**Comment by miguelgrinberg** [comment]
Please do not write issues on this project when you have questions. There is a discussion board for questions.

My guess is that `GET takepic/true` is a reference to an HTTP request that you need to make. What makes you think you have to use a Socket.IO connection for this?

Also, you appear to have the emit and the event handler in the same file, which makes no sense. The idea is for a server to communicate with a client or viceversa. If the emit is in the client, then the event handler has to be in the server.
